src/tests/async/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="../Directory.Build.props" />
+  <PropertyGroup>
+    <Features>$(Features);runtime-async=on</Features>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)RuntimeAsyncMethodGenerationAttribute.cs" />
+  </ItemGroup>
+</Project>
```

Comment:
I think switching all tests to experimental compiler is good for runtimelab, but in main repo we might need to limit this to async folder. 

I assume this will work in CI, right?